dds/idl/value_writer_generator.cpp

```diff
@@ -71,6 +71,14 @@ namespace {
       return "boolean";
     case AST_PredefinedType::PT_octet:
       return "byte";
+    case AST_PredefinedType::PT_any:
+    case AST_PredefinedType::PT_object:
+    case AST_PredefinedType::PT_value:
+    case AST_PredefinedType::PT_abstract:
+    case AST_PredefinedType::PT_void:
+    case AST_PredefinedType::PT_pseudo:
```

Comment:
Are these lines needed now that we have "default"?

Reply:
Maybe not? I don't have a build that complains about them in front of me, just the warning message.

Reply:
I'll remove them and see what happens.